test/selenium/print_test.js

```diff
@@ -3,6 +3,12 @@
 var webdriver = require('browserstack-webdriver');
 
 var runTest = function(cap, driver, target) {
+  // Goto the travis deployed site.
+  driver.get(target + '/?lang=de&layers=KML%7C%7Chttp:%2F%2Fopendata.utou.ch%2Furbanproto%2Fgeneva%2Fgeo%2Fkml%2FRoutes.kml');
+  // Accept the third party data
```

Comment:
Would it be possible to use the wait function here to check if an alert is there? Pseudo code:
```
driver.wait(function() {
    //check for exisance of alert (not sure how, verify), return true when found
    if (driver.switchTo().alert()) {
        return true;
    } else {
        return false;
    }
}, 3000).then(function() {
    driver.switchTo().alert().accept();
    ...

});
```